.github/workflows/docker.yml

```diff
@@ -3,12 +3,37 @@ name: Docker
 on:
   push:
     branches: [ main ]
-  pull_request: {}
+  pull_request:
+    branches: [ main ]
+  release:
+    types: [ published ]
```

Comment:
Have you tested (in another repo for example) that this actually works in combination with `docker/metadata-action`? I know that it was me who suggested doing it on release publish instead of tag push, but after looking at [the documentation](https://github.com/marketplace/actions/docker-metadata-action) which only ever mentions `event=tag` but never `release`, I'm not sure if this will just work (though I'd prefer it if it does).

Reply:
The tests I did for https://github.com/Icinga/icinga-notifications/pull/249#discussion_r1689324473 were using the release published event, so yes, it works.

Reply:
That's why I've also opted for ` tags: [ 'v*' ]`.

Reply:
> The tests I did for [#249 (comment)](https://github.com/Icinga/icinga-notifications/pull/249#discussion_r1689324473) were using the release published event, so yes, it works.

Missed your comment before posting my comment. I will use that as well in Icinga for Kubernetes then.